### ISSUE TITLE
Draft Card holders policy

### ISSUE THREAD

**Issue by lilianabermudes** [CodeYourFuture/CYF-Staff-Tickets#234]

**Comment by ClaireBickley** [comment]
Cynthia has started a [draft](https://docs.google.com/document/d/1_8jjmQE9imxupZlcxdUnW05h9tvlKmqjTvF-GyIOTFs/edit)

need to finalise staff expenses policy first

**Comment by lilianabermudes** [comment]
Thank you, Claire.

We're talking about Corporate cards - but maybe we need to clarify if they will be credit cards or pre-praid? Or both?

**Comment by ClaireBickley** [comment]
yes - this is the policy we asked Cynthia to write as part of her initial task. I don't think we'll go ahead with policy in it's current state, especially as there will only be one credit card. There will be pre-paid cards for some staff members who need them so we can adapt the policy to include all of that. But I think the first step is to put this on hold and focus on finalising the staff expenses policy and returning to this at a later date.

**Comment by cynthimicallef101** [comment]
Pre-Paid Cards Research:

WISE (CYF has a Wise Account)

1. They are called Employee Expense Cards, not strictly a pre-paid card
2. Cost: First card is free - does CYF already have a  Wise Debit or Credit card? Which one? Who has access to it? Subsequent cards cost £3 each, one-time fee
3. Spending limits: Can set daily or monthly spending limits. Same for ATM withdrawal limits.
4. Might not apply for CYF: These limits apply across one business account. If one team member reaches the daily limit for ATM withdrawals, other team members won’t be able to withdraw more money from the business account until the limit refreshes at midnight.
5. Connects with Xero 
6. Can upload receipts after paying with the card
7. There are no subscriptions or monthly fees. No transaction fees on foreign currencies.

HSBC Innovation Banking Business Charge Card

Control:
Set a maximum spending limit for each cardholder
Control permitted spending categories to ensure cards are only used for their intended purpose
Choose whether or not your cardholders have access to cash
- 56 days’ interest free credit
- System administrators can self-manage users and their access to services and accounts
- Control payment authorisation as either sole or dual control
- Set up profile and/or user daily authorisation payment limits

**Comment by LeandaCYF** [comment]
Hi Cynthia
thanks for the above research.

the Wise account cards described above are the cards we have had for employees up until now, they are debit cards linked directly to the bank which we want to move away from for security purposes.  
Do you know if wise do a credit card rather than the employee debit cards ? we are looking at having just 1 x credit card in finance.

Is the HSBC card described above a credit card ? (I am guessing yes as there is 56 days interest free credit) but please confirm. 

Have you looked into any pre paid card options at all ? 

Thanks
Leanda

**Comment by cynthimicallef101** [comment]
@LeandaCYF to decide and organise new card for CYF. Waiting for card decision to write the policy.

**Comment by AmyCYF** [comment]
- [ ] Elena set up group call to discuss this will Lili, Amy, Leanda and Amanda

**Comment by ClaireBickley** [comment]
now covered in Delegation of Authority Policy
